<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(254,254,254)</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>1371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font: 48pt &quot;HelveticaNeueCyr&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Корзина пуста</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>1371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font: 20pt&quot;HelveticaNeueCyr&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Авторизованные пользователи могут добавлять товары в корзину.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>670</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed {
    background-color: rgb(108,8,8);
}
QPushButton {
border: 2px solid #521;
border-radius: 10px;
border-style: outset;
background: rgb(128,24,24);
color:rgb(254,254,254);
font-weight: bold;
font: 26pt &quot;Futurespore Cyrillic&quot;;
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>1381</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QPushButton" name="prev">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>670</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed {
border: 0px solid #000;
border-radius: 10px;
border-style: outset;
background: black;
color: rgb(254,254,254);
font-weight: bold;
font: 22pt &quot;Futurespore Cyrillic&quot;;
}
QPushButton {
border: 2px solid #000;
border-radius: 10px;
border-style: outset;
color: black;
font-weight: bold;
font: 22pt &quot;Futurespore Cyrillic&quot;;
}</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>670</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed {
border: 0px solid #000;
border-radius: 10px;
border-style: outset;
background: black;
color: rgb(254,254,254);
font-weight: bold;
font: 22pt &quot;Futurespore Cyrillic&quot;;
}
QPushButton {
border: 2px solid #000;
border-radius: 10px;
border-style: outset;
color: black;
font-weight: bold;
font: 22pt &quot;Futurespore Cyrillic&quot;;
}</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>1379</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futurespore Cyrillic</family>
     <pointsize>110</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font: 110pt &quot;Futurespore Cyrillic&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Karmen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="korzina">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed {
border: 0px solid #000;
border-radius: 13px;
border-style: outset;
background: black;
color: rgb(254,254,254);
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}
QPushButton {
border: 2px solid #000;
border-radius: 13px;
border-style: outset;
color: black;
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>1379</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futurespore Cyrillic</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
color: black;
font: 26pt &quot;Futurespore Cyrillic&quot;;</string>
   </property>
   <property name="text">
    <string>Стиль, который говорит о тебе</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="like">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed {
border: 0px solid #000;
border-radius: 13px;
border-style: outset;
background: black;
color: rgb(254,254,254);
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}
QPushButton {
border: 2px solid #000;
border-radius: 13px;
border-style: outset;
color: black;
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="log_in">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed {
border: 0px solid #000;
border-radius: 13px;
border-style: outset;
background: black;
color: rgb(254,254,254);
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}
QPushButton {
border: 2px solid #000;
border-radius: 13px;
border-style: outset;
color: black;
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>1381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font: 20pt&quot;HelveticaNeueCyr&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Вам может понравиться:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>1381</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
